src/components/MemberRequestCard/MemberRequestCard.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import styles from './MemberRequestCard.module.css';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import { useMutation } from '@apollo/client';
+import {
+  ACCEPT_ORGANIZATION_REQUEST_MUTATION,
+  REJECT_ORGANIZATION_REQUEST_MUTATION,
+} from 'GraphQl/Mutations/mutations';
+interface MemberRequestCardProps {
+  key: string;
+  id: string;
+  memberName: string;
+  memberLocation: string;
+  joinDate: string;
+  memberImage: string;
+  email: string;
+}
+function MemberRequestCard(props: MemberRequestCardProps): JSX.Element {
+  const [addm] = useMutation(ACCEPT_ORGANIZATION_REQUEST_MUTATION);
+  const [removem] = useMutation(REJECT_ORGANIZATION_REQUEST_MUTATION);
```

Comment:
please use the full phrase for `addm` and `removem`

I can't tell if its to be `addMember` and `removeMember`

Or

`acceptMutation` and `rejectMutation`

If it is `acceptMutation` and `rejectMutation`, then it should be adjusted as follows:
```suggestion
  const [acceptMutation] = useMutation(ACCEPT_ORGANIZATION_REQUEST_MUTATION);
  const [rejectMutation] = useMutation(REJECT_ORGANIZATION_REQUEST_MUTATION);
```